client/phase.py

```diff
@@ -22,20 +22,21 @@
 
 global patience
 
+
 @click.command()
-@click.option('-r', '--remote-chain', default=None, help='choose one of the remote chains: gesell.')
-@click.option('--client', default='../target/release/encointer-client-notee', help='Client binary to communicate with the chain.')
+@click.option('--client', default='../target/release/encointer-client-notee',
+              help='Client binary to communicate with the chain.')
+@click.option('-u', '--url', default='ws://127.0.0.1', help='URL of the chain, or `gesell` alternatively.')
```

Comment:
I know that `-u` and `-r` were not the exact same thing (-r accepts `gesell`) as value and will look up the gesell url. However, I wanted to have a uniform interface. So everything is `-u` know, but I have documented it in help, if `gesell` is also an accepted value.

Reply:
yeah, that was a hack anyway...